aspnetcore/grpc/httpapi.md

```diff
@@ -67,7 +67,7 @@ Traditional gRPC over HTTP/2 supports streaming in all directions. Transcoding i
 
 Server streaming methods use [line-delimited JSON](https://wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON). Each message written using `WriteAsync` is serialized to JSON and followed by a new line.
 
-The following server streaming method writes 3 messages.
+The following server streaming method writes 3 messages:
```

Comment:
For a count, they recommend spelling out 0-9 ...

```suggestion
The following server streaming method writes three messages:
```

Cross-ref: https://docs.microsoft.com/style-guide/numbers